webapp-client/src/component/CategoryList.js

```diff
@@ -1,30 +1,74 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import ErrorMessage from './ErrorMessage';
 
 export default class CategoryList extends React.Component {
-  constructor(props){
+  constructor(props) {
```

Comment:
Maybe can we add a comment for what is expected in props? Do you think that's necessary?